frontend/src/app/Settings/Settings.tsx

```diff
@@ -13,26 +15,55 @@ const Settings: React.FC<RouteComponentProps<{}>> = ({ match }) => {
   return (
     <main className="prime-home">
       <div className="grid-container">
-        <SettingsNav />
-        <Route exact path={match.url} component={ManageOrganizationContainer} />
-        <Route
-          path={match.url + "/facilities"}
-          component={ManageFacilitiesContainer}
-        />
-        <Route
-          path={match.url + "/facility/:facilityId"}
-          render={({ match }: RouteComponentProps<Params>) => (
-            <FacilityFormContainer facilityId={match.params.facilityId} />
+        <PrimeErrorBoundary
```

Comment:
There's definitely a bigger discussion about how app errors are handled because we're definitely super inconsistent. I think there _should_ be at least one error boundary at a lower level than the full app. It's just less jarring when you don't lose the header/nav if you don't have to. But yeah, I made the choice to put the settings error handling all right here on my own, so I'm not sure what to do.